apps/src/music/SoundsPanel.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './soundsPanel.module.scss';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const getSoundRowClassName = (currentValue, folderPath, soundSrc) => {
+  if (currentValue === folderPath + '/' + soundSrc) {
+    return styles.soundRowSelected;
+  } else {
+    return styles.soundRow;
+  }
+};
+
+const getPreviewClassName = (playingPreview, folderPath, soundSrc) => {
+  if (playingPreview === folderPath + '/' + soundSrc) {
+    return styles.previewPlaying;
+  } else {
+    return styles.preview;
+  }
+};
+
+const getIcon = type => {
+  const typeToIcon = {
+    beat: 'volume-up',
+    bass: 'headphones',
+    lead: 'music',
+    fx: 'asterisk'
```

Comment:
It's not clear to me how these icons were chosen. Will there eventually be a key somewhere such as in the instructions panel?

Reply:
Oh it was a rather arbitrary set chosen quickly by @moneppo, and in fact we'd like to use more appropriate icons as soon as we upgrade our icon library.  I think we can expect this to evolve as we improve the interface.